lib/matplotlib/tests/test_dviread.py

```diff
@@ -1,34 +1,32 @@
-from matplotlib.testing.decorators import skip_if_command_unavailable
+import json
+from pathlib import Path
+import shutil
 
 import matplotlib.dviread as dr
-import os.path
-import json
 import pytest
 
 
 def test_PsfontsMap(monkeypatch):
     monkeypatch.setattr(dr, 'find_tex_file', lambda x: x)
 
-    filename = os.path.join(
-        os.path.dirname(__file__),
-        'baseline_images', 'dviread', 'test.map')
+    filename = str(Path(__file__).parent / 'baseline_images/dviread/test.map')
```

Comment:
Shouldn't `'baseline_images/dviread/test.map'` still be created with `os.path.join`?

Reply:
no, pathlib handles these internally just fine (see e.g. third example of https://docs.python.org/3/library/pathlib.html#operators).